src/contrib/mlir/runtime/cpu/cpu_runtime.hpp

```diff
@@ -38,7 +38,7 @@ namespace ngraph
                 void* allocatedPtr;
                 void* alignedPtr;
                 int64_t offset;
-                int64_t shapeAndStrides[];
+                std::vector<int64_t> shapeAndStrides;
```

Comment:
This is intended to be used as an input descriptor used in MLIR compiled code so I think data is expected to be statically allocated. It's probably not failing now because we are not generating code that uses this field, I think. Let's ask Amy since she run some tests with this.

Reply:
Didn't fail because callbacks are currently under an env variable. Maybe we should enable it by default ? 
@ayzhuang for reference